tests/metrics/process/test_hunks_count.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from pydriller.metrics.process.hunks_count import HunksCount
+
+TEST_DATA = [
+    ('https://github.com/ishepard/pydriller', 'scm/git_repository.py', '71e053f61fc5d31b3e31eccd9c79df27c31279bf', '71e053f61fc5d31b3e31eccd9c79df27c31279bf', 8),
```

Comment:
`'scm/git_repository.py'` should become `str(Path('scm/git_repository.py'))`

Reply:
Done, thanks.